README.md

```diff
@@ -0,0 +1,81 @@
+
+# Tarradino
+
+This is a port of Apogee's 3D action title Rise of the Triad, originally
+released in 1994. This port duplicates the functionality of the original
+game on modern operating systems, including Linux, Windows, and Mac OS X.
+
+Portability fixes include support for 64-bit and big-endian systems.
+
+This port makes use of the Simple Direct Media Layer, or SDL, and an additional
+library named SDL_mixer.
+
+## Known Issues
+
+- Demos go out of sync.
+- No netplay support.
+- Won't build with MSVC.
+- Some crashes in the sound mixer.
```

Comment:
I think I have solved the sound mixer crashes here?

https://github.com/fabiangreffrath/rott/commit/45a9203d8160ccf0c045568fa4b5b93cba1d6191

Reply:
I was getting a crash in my fork with the Excalibat at `FX_StopSound()` in `fx_mixer.c`. I couldn't figure out why, but somewhere along the way a value way higher than `MAX_CHANNELS` was getting passed into that function, causing an out of bounds memory access.

Reply:
Okay, that's a bug that I'd like to get reported, but probably not worth a generalized comment in the project's README. 😉

Reply:
> I was getting a crash in my fork with the Excalibat at `FX_StopSound()` in `fx_mixer.c`. I couldn't figure out why, but somewhere along the way a value way higher than `MAX_CHANNELS` was getting passed into that function, causing an out of bounds memory access.

I think the problem is that many `SD_Play*()` functions return `0` to report failure, whereas `0` is indeed a valid channel number.

Reply:
> > I was getting a crash in my fork with the Excalibat at `FX_StopSound()` in `fx_mixer.c`. I couldn't figure out why, but somewhere along the way a value way higher than `MAX_CHANNELS` was getting passed into that function, causing an out of bounds memory access.
> 
> I think the problem is that many `SD_Play*()` functions return `0` to report failure, whereas `0` is indeed a valid channel number.

This should be solved by now.